Makefile

```diff
@@ -316,6 +318,14 @@ shell/public/email-494949.svg: icons/email.svg
 	@$(call color,custom color $<)
 	@sed -e 's/#111111/#494949/g' < $< > $@
 
+shell/public/question-a9a9a9.svg: icons/question.svg
```

Comment:
Should we be adding this to the fonticons set?  I seem to remember us wanting to move away from manually recoloring a bunch of SVGs and using CSS `color:` instead.

OTOH, it's easier to get exact pixel sizes right with `background-image`.  So don't consider this a blocker.